etc/tokens.json

```diff
@@ -484,5 +484,14 @@
     "decimals": 8,
     "logo": "https://github.saturn.network/etc/logo/lct_2.png",
     "forum": "https://forum.saturn.network/t/placeholder-for-something-new-d/3157" 
+  },
+  {
+    "name": "Candy Classic",
+    "address": "0x0b6f824fab88281ea8bd883bdfc1f586743f92fa",
+    "symbol": "CANDY",
+    "decimals": 6,
+    "logo": "https://github.saturn.network/etc/logo/candy final.png",
+    "website": "",
```

Comment:
If you do not have a website please just remove this line instead of supplying an empty string.